frontend/pages/hosts/details/cards/Software/SelfService/SelfServiceItem/_styles.scss

```diff
@@ -0,0 +1,97 @@
+.self-service-item {
+  &__item {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__item-content {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  &__item-topline {
+    display: flex;
+    flex-direction: row;
+    height: 64px;
+    align-items: center;
+    gap: 16px;
+    overflow: hidden;
+  }
+
+  &__item-icon {
+    display: flex;
+    height: 64px;
+    min-width: 64px;
+  }
+
+  &__item-name-version {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 64px;
+    overflow: hidden;
+  }
+
+  &__item-name {
+    font-size: $x-small;
+    font-weight: $bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    text-wrap: nowrap;
```

Comment:
Do we separately need to add a tooltip with the untruncated values for name and version?